src/main/java/io_utilities/working_with_input/FormatChecker.java

```diff
@@ -0,0 +1,59 @@
+package io_utilities.working_with_input;
+
+import enums.CommandFormats;
+import exceptions.WrongInputFormatException;
+
+import java.util.HashMap;
+
+public class FormatChecker {
+    private final HashMap<String, CommandFormats> commandFormatsHashMap = new HashMap<>();
+
+    public FormatChecker() {
+    }
+
+    public void init() {
+        registerCommandFormat("add", CommandFormats.WITHOUT_ARG);
+        registerCommandFormat("clear", CommandFormats.WITHOUT_ARG);
+        registerCommandFormat("execute_script", CommandFormats.WITH_STRING_ARG);
+        registerCommandFormat("exit", CommandFormats.WITHOUT_ARG);
+        registerCommandFormat("filter_contains_name", CommandFormats.WITH_STRING_ARG);
+        registerCommandFormat("help", CommandFormats.WITHOUT_ARG);
+        registerCommandFormat("info", CommandFormats.WITHOUT_ARG);
+        registerCommandFormat("min_by_coordinates", CommandFormats.WITHOUT_ARG);
+        registerCommandFormat("print_field_ascending_house", CommandFormats.WITHOUT_ARG);
+        registerCommandFormat("remove_by_id", CommandFormats.WITH_NUMERAL_ARG);
+        registerCommandFormat("remove_first", CommandFormats.WITHOUT_ARG);
+        registerCommandFormat("remove_lower", CommandFormats.WITHOUT_ARG);
+        registerCommandFormat("save", CommandFormats.WITHOUT_ARG);
+        registerCommandFormat("show", CommandFormats.WITHOUT_ARG);
+        registerCommandFormat("sort", CommandFormats.WITHOUT_ARG);
+        registerCommandFormat("update", CommandFormats.WITH_NUMERAL_ARG);
```

Comment:
I've already wrote comment about this above, make these a part of the command class